core/query/src/main/java/org/eclipse/rdf4j/query/explanation/GenericPlanNode.java

```diff
@@ -217,14 +217,25 @@ public void setAlgorithm(String algorithm) {
 		this.algorithm = algorithm;
 	}
 
+	private static int prettyBoxDrawingType = 0;
+
 	/**
 	 * Human readable string. Do not attempt to parse this.
 	 *
 	 * @return
 	 */
 	@Override
 	public String toString() {
+		return getHumanReadable(0);
```

Comment:
Can you remind me - this is separate from what I would get if I did 

     TupleQuery query = conn.prepareTupleQuery(....);
     System.out.println(query);

correct? What I'm getting at is that this influences only the output of the `Explanation` serialization, not if we directly serialize a query algebra node to string. If so: that's good, I'd rather not change the original tupleExpr serialization too much (even if it's not, strictly speaking, a standardized format).

Reply:
Yes. Printing the query like that just prints the parsed version of the tuple expression. Using the explanation transforms the query into GenericPlanNode, where we have this bow-drawing toString addition.